pkg/cmdline/cmdline.go

```diff
@@ -156,14 +157,21 @@ func progress(cli *cmdline, app, desc string, codes <-chan builder.SummaryStatus
 	}
 }
 
-func passStr(msg string) string {
-	const pass = "SUCCESS " + "⚓"
-	return fmt.Sprintf("%s: %s", green(msg), pass)
+func passStr(msg string, disableEmoji bool) string {
+	return fmt.Sprintf("%s: %s", green(msg), concatStrAndEmoji("SUCCESS", " ⚓ ", disableEmoji))
 }
 
-func failStr(msg string) string {
-	const fail = "FAIL " + "❌"
-	return fmt.Sprintf("%s: %s", red(msg), fail)
+func failStr(msg string, disableEmoji bool) string {
+	return fmt.Sprintf("%s: %s", red(msg), concatStrAndEmoji("FAIL", " ❌ ", disableEmoji))
+}
+
+func concatStrAndEmoji(text string, emoji string, disableEmoji bool) string {
+	var concatStr bytes.Buffer
```

Comment:
more of an FYI, but you can use a [`strings.Builder`](https://golang.org/pkg/strings/#Builder) here instead for more efficient memory usage.

Reply:
Thanks, it was introduced in 1.10 right? Gotcha